PORTAL/jobs/__main__.py

```diff
@@ -729,8 +731,8 @@ def _add_request_cli(add_cmd: Callable, add_hidden: bool = True) -> Callable:
                                help='do PGO and LTO (default)')
     compile_bench.add_argument('--no-optimize', dest='optimize', action='store_false')
     compile_bench.add_argument('--debug', action='store_true')
-    compile_bench.add_argument('--benchmarks')
-    compile_bench.add_argument('--remote', help='(default: origin)')
+    compile_bench.add_argument('--benchmarks', help='list of benchmarks to run (default: all)')
+    compile_bench.add_argument('--remote', help='(default: python)')
```

Comment:
What is "python" as a git remote?  Is "origin" confusing or not correct?

Reply:
Presumably 'origin' is confusing because people have used it for different purposes, led on by various tutorials. As remotes are usually named after the user owning the fork, it makes sense to rename the origin to the user or project owning the upstream repo.

Reply:
Yes -- I sort of saw "origin" as a generic term, "python" as a little more specific about the owner of the fork that's being talked about.